thinc/util.py

```diff
@@ -19,7 +19,8 @@
 
 DATA_VALIDATION: ContextVar[bool] = ContextVar("DATA_VALIDATION", default=False)
 
-from .types import ArrayXd, ArgsKwargs, Ragged, Padded, FloatsXd, IntsXd  # noqa: E402
+
+from .types import ArrayXd, ArgsKwargs, Ragged, Padded, FloatsXd, IntsXd, Floats2d # noqa: E402
```

Comment:
I don't know what happened here historically, but let's move the `DATA_VALIDATION` definition to below all the imports, and move the very last `from typing` import to the second line.

(this is outside your PR but just happened to notice this, so we might as well clean it up)